drivers/vmwareworkstation/vmrun.go

```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright 2014 VMware, Inc.  All rights reserved.  Licensed under the Apache v2 License.
+ */
+
+package vmwareworkstation
+
+import (
+	"bytes"
+	"errors"
+	"fmt"
+	"os"
+	"os/exec"
+	"strings"
+
+	"github.com/docker/machine/log"
+)
+
+// TODO un-hardcode these locations
```

Comment:
You could use an `exec.LookPath` for this which falls back to a `os.Stat` of the hardcoded paths to check if they're around.